src/java/lang.rs

```diff
@@ -5,34 +5,29 @@ use crate as duchess;
 
 use crate::java;
 
-duchess_macro::duchess_javap! {
-r#"
-        Compiled from "Object.java"
-        public class java.lang.Object {
-            public java.lang.Object();
-                descriptor: ()V
-
-            public final native java.lang.Class getClass();
-                descriptor: ()Ljava/lang/Class;
-
-            public native int hashCode();
-                descriptor: ()I
-
-            public boolean equals(java.lang.Object);
-                descriptor: (Ljava/lang/Object;)Z
-
-            public java.lang.String toString();
-                descriptor: ()Ljava/lang/String;
-
-            public final native void notify();
-                descriptor: ()V
-
-            public final native void notifyAll();
-                descriptor: ()V
+// Declare the class "object" in isolation.
+// Eventually we'd like to move all the declarations
+// into a `java_package` call, but we still have
+// to fix some bugs, so for now, just do that call
+// inside of a `mod object` and re-export what we want.
+mod object {
+    use super::*;
+    duchess_macro::java_package! {
+        package java.lang;
+
+        class Object {
+            Object();
+            hashCode();
+            equals(Object);
+            toString();
+            notify();
+            notifyAll();
```

Comment:
1. Yes, that's a bug in the PR, it should be `java.lang.Object`.
2. Yes.
3. I expect to do that eventually.

Reply:
```suggestion
            hashCode();
            equals(java.lang.Object);
            toString();
            notify();
            notifyAll();
```

Reply:
Regarding 1: In a follow-up PR, I have code that checks each method signature and makes sure that it matches *something*, so it would have caught this case and reported an error.